airflow/decorators/base.py

```diff
@@ -309,6 +309,8 @@ def map(self, **kwargs) -> XComArg:
             partial_kwargs.get("retry_delay", DEFAULT_RETRY_DELAY),
         )
         partial_kwargs.setdefault("executor_config", {})
+        partial_kwargs.setdefault("inlets", [])
+        partial_kwargs.setdefault("outlets", [])
```

Comment:
Should these be None?

Reply:
Or not. I was wrong and these aren't needed at all actually.